src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/CachePoolPass.php

```diff
@@ -26,34 +26,18 @@ class CachePoolPass implements CompilerPassInterface
     public function process(ContainerBuilder $container)
     {
         foreach ($container->findTaggedServiceIds('cache.pool') as $id => $tags) {
-            $pool = $container->getDefinition($id);
+            $adapter = $pool = $container->findDefinition($id);
```

Comment:
`getDefinition()` is sufficient as `findTaggedServiceIds()` does not return aliases